CHANGELOG.md

```diff
@@ -19,6 +19,8 @@ it in future.
 * **Corrections in MuonDIS simulation**
   The DIS interactions are now time-shifted to be consistent with the original incoming muon. Additionally, tracks from soft interactions of the original muon along with the muon's veto response are preserved (in muonDis.root) and included up to the DIS interaction point. To be noted that the muon veto points are manually added using add_muonresponse.py, which modifies the simulation file. This replaces the old method of "backward-travelling muon" to generate the incoming muon's veto response. All MuonDIS simulation scripts have been updated and consolidated within FairShip/muonDIS, ensuring consistency for new productions.
 * Added a custom CrossSection branch to the simulation file to save the DIS cross sections from muonDIS.
+* New Optimized version for MS in geometry_config
+* Added new parameters to MS definition
```

Comment:
This is very vague. It might be worth mentioning which parameters, if users should know. Otherwise something like "Added new parameters to muon shield to support configurations from current optimisation campaign" would be clearer.